### ISSUE TITLE
Are you planning on filling this out or is this just a placeholder?

### ISSUE THREAD

**Issue by deten** [MythikGN/EasyLoA#1]
Hi,

I have been coming back to check this out occasionally and haven't seen any work done. I was curious if you were planning on doing any or if this was just a placeholder for something else?

Thanks!

**Comment by DarkLotus** [comment]
I lost interest in Loa as it moved further from uo. May come back to it one
day

**Comment by simon3167** [comment]
Have you tried the community server Legends of Ultima? If not, go for it... you will enjoy it.

**Comment by DarkLotus** [comment]
Not Ultima in any sense of the word sadly I'm glad their tryingGbut Lou is
nothing like uo sadly

**Comment by Lady-Binary** [comment]
People might be interested in this project: https://github.com/Lady-Binary/EasyLOU

I understand you are not maintaining EasyLOA anymore. Feel free to help out with this new one.

Enjoy :)

**Comment by DarkLotus** [comment]
Cool project, without extending the easyuo language it doesn't really seem
like easyuo though and more a general assistant. Glad to see some people
still having an interest in Loa, wish things had progressed differently and
it was closer to the uo vibes we would have all liked rather than dead.on
arrival.

**Comment by Lady-Binary** [comment]
I've chosen to adopt LUA as a scripting language rather than relying on the EasyUO language - above all, to be more "standard", and having multiple parsers at my disposal should Moonsharp not fit my needs.

Have you looked at https://github.com/Lady-Binary/EasyLOU/blob/master/COMMANDS.md and at https://github.com/Lady-Binary/EasyLOU/blob/master/VARIABLES.md? I'm not sure what you mean with being a general assistant rather than EasyUO. You can do pretty much anything you could do with EasyUO (with some things not implemented yet - like GUI capabilities and Hotkeys). If you have any suggestion or any feedback at all on how I've implemented things, I'm listening!

Thank you for looking at it!

**Comment by DarkLotus** [comment]
Yeah, lua is a much nicer language with a wider support base so it's a good
choice. Your welcome to use the name easyloa as well I don't have any right
to it and I gave up on loa months ago. May come back if they ever lift the
nda.
